tensilelite/Tensile/KernelWriterModules.py

```diff
@@ -319,11 +319,12 @@ def moveMIoutToArch(kernel, startVgprAlphaTmp):
       imod.itemList[destIdx] = VMovB32(dst=vgpr(Holder(name="ValuC")),
                                                      src=vgpr("ValuC+%u"%srcIdx), comment="Rearrange MI out reg")
     elif kernel["ProblemType"]["ComputeDataType"].isSingleComplex():
-        accImOffset = accVgprImagNumOffset(kernel, lrvwB)
-        cimod = Module()
-        cimod.add(VMovB32(dst=vgpr(Holder(name="ValuC")), src=vgpr("ValuC+%u"%srcIdx), comment="Rearrange MI out reg"))
-        cimod.addInst(VMovB32(dst=vgpr(Holder(name="ValuC+1")), src=vgpr("ValuC+%u"%(srcIdx+accImOffset)), comment="Rearrange MI out reg"))
-        imod.itemList[destIdx] = cimod
+        # accImOffset = accVgprImagNumOffset(kernel, lrvwB)
+        # cimod = Module()
+        # cimod.add(VMovB32(dst=vgpr(Holder(name="ValuC")), src=vgpr("ValuC+%u"%srcIdx), comment="Rearrange MI out reg"))
+        # cimod.addInst(VMovB32(dst=vgpr(Holder(name="ValuC+1")), src=vgpr("ValuC+%u"%(srcIdx+accImOffset)), comment="Rearrange MI out reg"))
+        # imod.itemList[destIdx] = cimod
+        raise NotImplementedError
     elif kernel["ProblemType"]["ComputeDataType"].isDoubleComplex():
       accImOffset = accVgprImagNumOffset(kernel, lrvwB)
```

Comment:
Should this part be removed?

Reply:
The PR owner only focus only where use addInst()